package.json

```diff
@@ -6,7 +6,7 @@
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "jest",
     "tsc": "tsc -p . --noEmit",
-    "lint": "tslint -c tslint.json --project ."
+    "lint": "eslint --fix --ext .mjs,.js,.jsx,.ts,.tsx ."
```

Comment:
Ok so part of our CI is to call `yarn lint` to make sure it passes all the lint rules. Wondering if we should remove `--fix` here since the point is to test if linting passes. If a developer wants to force fixes to run, they can locally run `yarn lint -- --fix`?

Reply:
Shouldn't have any impact, as the code should already be fixed by the time it reaches CI. Without --fix, then a autofixable linting error would cause a failure, which is a trivial thing causing a build failure.